merlin

```diff
@@ -9,7 +9,12 @@ ini_set('memory_limit', -1);
  * Application entrypoint for the migration framework.
  */
 
-require __DIR__ . '/vendor/autoload.php';
+foreach ([__DIR__ . '/vendor/autoload.php', __DIR__ . '/../../../vendor/autoload.php'] as $file) {
```

Comment:
```suggestion
foreach ([__DIR__ . '/../../../vendor/autoload.php', __DIR__ . '/../../autoload.php', __DIR__ . '/../vendor/autoload.php', __DIR__ . '/vendor/autoload.php'] as $file) {
```

https://github.com/sebastianbergmann/phpunit/blob/master/phpunit#L30 does the same type of thing for bundled phars, so for completeness we should probably do the same thing.